doc/tntcxx_getting_started.rst

```diff
@@ -0,0 +1,524 @@
+use the `example application <https://github.com/tarantool/tntcxx/blob/master/examples/Simple.cpp>`_
+from the connector repository. We will go step by step through the application
+code and explain what each part does.
+
+The following main topics are discussed in this manual:
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _gs_cxx_prereq:
+
+Pre-requisites
+----------------
+
+To go through this Getting Started exercise, we need the following
+pre-requisites to be done:
+
+* :ref:`installing and building the Tarantool C++ connector <gs_cxx_prereq_install>`
+* :ref:`starting Tarantool and creating a database <gs_cxx_prereq_tnt_run>`
+* :ref:`seting up access rights <gs_cxx_prereq_access>`.
+
+.. _gs_cxx_prereq_install:
+
+Installation
+~~~~~~~~~~~~~
+
+Currently supported OS for the Tarantool C++ connector usage is Linux.
+To install the connector in your OS, you should build it from the source.
+
+.. //TBD For the 3d-party  and Tarantool steps, maybe later we can move these details to REAMDE and leave here the links to those sections of README
+
+#. Make sure you have the necessary third-party software. If you miss some of
+   the items, install them:
+
+   * `install <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+      ``git``, a version control system
+   * `install <https://linuxize.com/post/how-to-unzip-files-in-linux/#installing-unzip>`_
+      the ``unzip`` utility
+   * `install <https://gcc.gnu.org/install/>`_ the ``gcc`` compiler `complied
+      with the C++17 standard <https://gcc.gnu.org/projects/cxx-status.html#cxx17>`_.
+   * `install <https://cmake.org/install/>`_ the ``cmake`` and ``make`` tools.
+
+#. If you don't have Tarantool in you OS, install it in one of the ways:
+
+   * from a package--refer to `OS-specific instructions <https://www.tarantool.io/en/download/>`_
+   * from the `source <https://www.tarantool.io/en/download/os-installation/building-from-source/>`_.
+
+#. Clone the Tarantool C++ connector repository and build the connector from
+   the source.
+
+   .. code-block:: bash
+
+      git clone git@github.com:tarantool/tntcxx.git
+      cd tntcxx
+      cmake .
+      make
+
+.. _gs_cxx_prereq_tnt_run:
+
+Starting Tarantool and creating a database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Start Tarantool :ref:`locally <getting_started-using_package_manager>`
+or :ref:`in Docker <getting_started-using_docker>`
+and create a space with the following schema and index:
+
+.. code-block:: lua
+
+    box.cfg{listen = 3301}
+    t = box.schema.space.create('t')
+    t:format({
+             {name = 'id', type = 'unsigned'},
+             {name = 'a', type = 'string'},
+             {name = 'b', type = 'number'}
+             })
+    t:create_index('primary', {
+             type = 'hash',
+             parts = {'id'}
+             })
+
+.. IMPORTANT::
+
+   Please do not close the terminal window where Tarantool is running.
```

Comment:
“Please” is for cases when we need something and ask the user/reader to do it.

> Please tell us about your experience with the C++ connector.
```suggestion
   Do not close the terminal window where Tarantool is running.
```

Reply:
Corrected.